### DISCUSSION TITLE
Artillery Stuck on 'Waiting for workers to come online...'

### DISCUSSION THREAD

**Discussion by samhunter1** [artilleryio/artillery#3349] [Artillery]
Hi, I'm using Artillery and Fargate to run load tests. Finding it hard to properly configure the AWS environment - I've followed all of the setup on the docs but the workers are still not coming online. I believe it might be a permissions issue but not sure how to pin point it as the logging on the AWS side is minimal.

Including the Artillery cluster details as well and there aren't any tasks or any log streams in the Artillery log group. 

![image](https://github.com/user-attachments/assets/7144bc36-0e2a-4ee0-a4ea-a9cb2be2e1a4)

_Edit (Adding for further context):_

This is the terminal, to show where abouts Artillery is getting stuck: 
![image](https://github.com/user-attachments/assets/f21763b5-b1b1-4aa2-b179-0f8ecf7989b0)

This is my ECS cluster list which doesn't appear to have any tasks: 

![image](https://github.com/user-attachments/assets/240206ca-0f33-4850-a577-28159e37c9ab)

One last question - as someone fairly new to AWS - what should I expect to see concerning clusters and tasks when I'm running Fargate with Artillery?

Any help is appreciated!

**Comment by malekerramli** [comment]
any updates here ? same issue
